tests/e2e-pw/utils/devtools.ts

```diff
@@ -16,26 +16,26 @@ const saveDevToolsSettings = async ( page: Page ) => {
 const getIsCardTestingProtectionEnabled = ( page: Page ) =>
 	page.getByLabel( 'Card testing mitigations enabled' ).isChecked();
 
-const toggleCardTestingProtection = ( page: Page ) =>
+const setCardTestingProtection = ( page: Page, enabled: boolean ) =>
```

Comment:
I changed this (and the other toggled function) because I don't trust toggles, if someone forgets the state as it shouldn't be, the other tests will start to fail too, making debugging the source of it harder. Setting the value explicitly will be in our favor here.